frontend/zc_messaging/src/pages/MessageBoard/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
```

Comment:
i thought message board is a component from zuri UI, why do we have it as a page here cos we don't have any page with /messageboard

Reply:
The MessageBoard component is the page that deals with messaging for the plugin. It should house the ZuriUi MessageBoard component. It won't be okay to use the ZuriUi MessageBoard component as the messaging page directly.

Reply:
Yeah, we can't just have the component from zuri UI like that. so I was thinking of having something like this

DM - /dm/:id
Channel - /ch/:id
for separation of concerns ... and both pages can just share components including the zuri UI component